nl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+
+"WELCOME_UPSELL" = "Welkom bij Broadcasts! In deze gratis versie kunnen we enkele voorkeurszenderes van bij ons toevoegen waarmee u uw bibliotheek kan starten. Of, als u liever upgradet naar de betaalde versie, kunt u uw eigen stations handmatig toevoegen of kiezen uit duizenden zenders van de lijst.";
+
+"WELCOME_ADD_PRESETS" = "Voeg voorkeuren toe";
+"WELCOME_NO_THANKS" = "Nee, bedankt";
+
+"GENERATED_COLLECTION_TITLE" = "Mijn verzameling";
+
+"UPSELL_COPY" = "Bedankt om Broadcasts te gebruiken!\n\nAls u dit wilt, kunt u de geavanceerde functies van Broadcasts ontgrendelen met een in-app-aankoop.\n\nOf, als u dit wilt, kunt u de vooraf ingestelde zenders blijven gebruiken en bewerken!";
+"UPSELL_BUY" = "Koop (%@)";
+"UPSELL_NOT_AVAILABLE" = "Niet beschikbaar";
+"UPSELL_RESTORE" = "Herstel aankopen";
+"UPSELL_NO_THANKS" = "Nee, bedankt";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "Nieuwe verzameling";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "Voeg zender toe";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "Toon verzamelingen";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "Toon of verberg navigatiekolom";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "Wissel tussen lijst- of rasterweergave";
+
+"IMAGE_WELL_REMOVE" = "Verwijder";
+"IMAGE_WELL_PASTE" = "Plak";
+"IMAGE_WELL_COPY" = "Kopieer";
+
+"CARPLAY_EMPTY_LIBRARY" = "Geen verzamelingen";
+"CARPLAY_ALL_STATIONS" = "Alle zenders";
+
+"MAC_ADD_STATION_BUTTON" = "Voeg zender toe";
+"MAC_EDIT_STATION_BUTTON" = "Wijzig zender";
+
+"MAC_ADD_COLLECTION_BUTTON" = "Voeg verzameling toe";
+"MAC_EDIT_COLLECTION_BUTTON" = "Wijzig verzameling";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "Zenderbrowser";
+"STATION_DIRECTORY_COUNTRIES_TITLE" = "Landen";
+"STATION_DIRECTORY_STATIONS" = "%@ zender";
+"STATION_DIRECTORY_STATIONS_PLURAL" = "%@ zenders";
+
+"STATION_DIRECTORY_NO_CONNECTION" = "Geen verbinding";
+"STATION_DIRECTORY_CONNECTING" = "Laden…";
+
+"STATION_DIRECTORY_TOOLBAR_ADD_MANUALLY" = "Voeg handmatig toe";
+"STATION_DIRECTORY_TOOLBAR_DONE" = "Klaar";
+
+"STATION_DIRECTORY_COUNTRIES_SEARCH_PLACEHOLDER" = "Zoek landen";
+"STATION_DIRECTORY_STATIONS_SEARCH_PLACEHOLDER" = "Zoek zenders";
+
+"ADD_STATION_CONTEXT_ADD_STATION" = "Voeg handmatig toe";
+"ADD_STATION_CONTEXT_SHOW_DIRECTORY" = "Voeg toe van de lijst";
+
+"LIBRARY_STATE_UPLOADING" = "Uploaden…";
+"LIBRARY_STATE_DOWNLOADING" = "Downloaden…";
+"LIBRARY_STATE_PREPARING" = "Voorbereiden…";
+"LIBRARY_STATE_READY" = "Bijgewerkt";
+"LIBRARY_STATE_MIGRATING" = "Migreren…";
+"LIBRARY_STATE_LOCALONLY" = "Alleen lokaal";
+
+"TV_TAB_ALL_STATIONS" = "Alle zenders";
+"TV_TAB_RECENTS" = "Onlangs afgespeeld";
+"TV_TAB_COLLECTIONS" = "Verzamelingen";
+
+"TV_NO_LIBRARY_MESSAGE" = "Begin uw Broadcasts-bibliotheek op uw iPhone, iPad of Mac en het zal gesynchroniseerd worden naar uw Apple TV.";
```

Comment:
"Begin een Broadcasts-bibliotheek op uw iPhone, iPad of Mac en deze zal ook gesynchroniseerd worden naar uw Apple TV."